experimental/oak_baremetal_kernel/src/mm/mod.rs

```diff
@@ -35,11 +37,12 @@ pub fn init<const N: usize, E: E820Entry>(
         PhysFrame::from_start_address(PhysAddr::new((N as u64 * 64 - 1) * Size2MiB::SIZE)).unwrap(),
     ));
 
+    /* Step 1: mark all RAM as available (event though it may contain data!) */
     memory_map
         .iter()
         .inspect(|e| {
             info!(
-                "E820 entry: [{:#016x}..{:#016x}) ({}), type {}",
+                "E820 entry: [{:#018x}..{:#018x}) ({}), type {}",
```

Comment:
Why did `016x` change?

Reply:
64-bit addresses are 16 hexadecimal digits long, but I forgot to account for the `0x` in front of them.